addons/sale/views/sale_order_views.xml

```diff
@@ -561,7 +561,7 @@
                                     <group invisible="display_type">
                                         <label for="customer_lead"/>
                                         <div name="lead">
-                                            <field name="customer_lead" class="oe_inline"/> days
+                                            <field name="customer_lead" class="oe_inline" readonly="product_type == 'service'"/> days
```

Comment:
```suggestion
                                            <field name="customer_lead" class="oe_inline" readonly="product_type != 'consu'"/> days
```

Please use the same condition everywhere, what if another module adds another type? :thinking: